docs/usage/plotting.rst

```diff
@@ -0,0 +1,40 @@
+.. _usage-plotting:
+
+Plotting
+========
+
+Boost-histogram does not contain plotting functions - this is outside of the
+scope of the project, which is histogram filling and manipulation. However, it
+does follow ``PlottableProtocol``, as listed below. Any plotting library that
+accepts an object that follows the ``PlottableProtocol`` can plot boost-histogram
+objects.
+
+Using the protocol:
+
+Plotters should only depend on the methods and attributes listed below. In short, they are:
+
+* ``h.kind``: The bh.Kind of the histogram (COUNT or MEAN)
+* ``h.values()``: The value (as given by the kind)
+* ``h.variances()``: The variance in the value (None if an unweighed histogram was filled with weights)
```

Comment:
This wording is not correct, because .variances() does not give you the variance of the mean if kind == MEAN it gives the variances of the samples. You need to divide that by .counts() to get the variance of the mean.

Reply:
I thought we decided in the PlottableProtocol that `.variances()` is always the variance of `.value()`, so that a "dumb" plotter could just plot values and variances and ignore the `.kind` and get something reasonable. It is also simpler if ".values" is the value of the Kind, and the ".variances" is the variances of the Kind. So in this case, the wording is correct, but the value is incorrect, it should be calculated as `.view()["variance"] / .view()["count']`? And a user wishing to get the sample variance would multiply by the `.counts()`?

I could easily be wrong here, though, either/both about what we decided and what is best. @jpivarski what does Uproot return for `.variances()` here for a TProfile?